models/webhook_msteams.go

```diff
@@ -0,0 +1,703 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
2019

Reply:
Updated in 15b22b02da578d547b87c82b9139b4f8733ec76d thanks